src/common/SwipeableFlatList/index.js

```diff
@@ -25,47 +16,26 @@ import { FlatList, View } from 'react-native';
 import PureSwipeableRow from './PureSwipeableRow';
 
 type SwipeableListProps = {
-  /**
-   * To alert the user that swiping is possible, the first row can bounce
-   * on component mount.
-   */
   bounceFirstRowOnMount: boolean,
-  // Maximum distance to open to after a swipe
   maxSwipeDistance: number | (Object => number),
-  // Callback method to render the view that will be unveiled on swipe
-  renderQuickActions: renderItemType,
   preventSwipeRight?: boolean,
+  renderQuickActions: renderItemType,
 };
 
 type Props<TEntity> = SwipeableListProps & FlatListProps<TEntity>;
 
 type State = {
   openRowKey: ?string,
+  refreshing: boolean,
```

Comment:
Can this be `isRefreshing`?

Reply:
ofc, just FlatList native prop called `refreshing` so I did it for consistency, but like you want )